src/launch/service.c

```diff
@@ -398,20 +413,19 @@ static int service_watch_unit_handler(sd_bus_message *message, void *userdata, s
         return 0;
 }
 
-static int service_watch_unit(Service *service, const char *unit) {
-        _c_cleanup_(c_freep) char *object_path = NULL;
+static int service_watch_unit_callback(sd_bus_message *message, void *userdata, sd_bus_error *errorp) {
```

Comment:
Maybe `service_watch_unit_load_handler()` to be consistent with our naming-scheme?